sgv2-service-common/src/main/java/io/stargate/sgv2/common/grpc/StargateBridgeClient.java

```diff
@@ -47,6 +48,25 @@ default Response executeQuery(Query query) {
     return Futures.getUninterruptibly(executeQueryAsync(query));
   }
 
+  // TODO create async variant?
+  Response executeQuery(
+      String keyspaceName, Function<Optional<CqlKeyspaceDescribe>, Query> queryProducer);
```

Comment:
I started with a synchronous implementation, because it is more straightforward, and what the REST service uses.

If we move forward with this, I'll convert it to async like the other methods, and probably create batch variants too.